### PR TITLE
[pipelineX](fix) Fix local exchange on pipelineX engine

### PR DESCRIPTION
**Pull request by Gabriel39**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Gabriel39** [comment]
run buildall

**Review by HappenLee — DISMISSED** [review]
LGTM

File: be/src/pipeline/exec/join_probe_operator.h [hunk 1]
```diff
@@ -50,15 +50,15 @@ class JoinProbeLocalState : public PipelineXLocalState<DependencyType> {
     // output expr
     vectorized::VExprContextSPtrs _output_expr_ctxs;
     vectorized::Block _join_block;
-    vectorized::MutableColumnPtr _tuple_is_null_left_flag_column;
-    vectorized::MutableColumnPtr _tuple_is_null_right_flag_column;
+    vectorized::MutableColumnPtr _tuple_is_null_left_flag_column {nullptr};
```
Comment (yiguolei) [inline be/src/pipeline/exec/join_probe_operator.h:53]: = nullptr

**Comment by Gabriel39** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by Gabriel39** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**